docs/telemetry.md

```diff
@@ -1,99 +1,269 @@
 # Telemetry
 
-Graphistry services can be configured to export telemetry information using the [OpenTelemetry](https://opentelemetry.io/) standard. This includes logs, metrics, and traces.
+## Overview
 
-When enabled, telemetry sent to the local Graphistry [OTEL Collector](https://opentelemetry.io/docs/collector/) service, and from there, dispatches to any observability tool that is compatible with the OpenTelemetry standard (e.g. Prometheus, Jaeger, Grafana Cloud, etc.).
+Graphistry services export telemetry information (metrics and traces) using the [OpenTelemetry](https://opentelemetry.io/) standard.
 
-To use telemetry with Graphistry, you need to:
+Graphistry services push their telemetry data to the [opentelemetry-collector](https://opentelemetry.io/docs/collector/) service (alias `otel-collector`) and this will forward the data to any observability tool that is compatible with the OpenTelemetry standard (e.g. Prometheus, Jaeger, Grafana Cloud, etc.).
 
-* Enable telemetry collection recording in one or more Graphistry services
-* Configure which users each service will monitor (see section on feature flags)
-* Enable the telemetry collector service
-* Configure the forwarding from the telemetry collector to an observability database/UI
-* Optionally, setup a local observability database/UI
+## Telemetry Deployment Modes in Graphistry
+
+When telemetry services are enabled, the OpenTelemetry Collector will be included in all deployment scenarios:
+
+### Forwarding to External Services
+In this mode, Graphistry forwards telemetry data to external services compatible with the OpenTelemetry Protocol (OTLP), such as Grafana Cloud.  The deployment will include only the OpenTelemetry Collector, which handles the data forwarding.
+
+### Using Packaged Observability Tools
+When using the local observability tools bundled with Graphistry —Prometheus, Jaeger, and Grafana— the OpenTelemetry Collector exports data to these services.  In this mode, you can configure Caddy to expose the observability tools to allow external access; the OpenTelemetry Collector URL won't be affected by Caddy (see [behind Caddy](#caddyfile---reverse-proxy-set-up)).  By default, the data is ephemeral, stored inside the containers, and won’t persist across restarts.  To enable data persistence, you can create a custom Docker Compose file named `custom_telemetry.yml` to configure persistent storage and update the environment variable `OTEL_COMPOSE_FILE` in the configuration file (`$GRAPHISTRY_HOME/data/config/telemetry.env`) to point to your `custom_telemetry.yml`.
```

Comment:
what goes into `custom_telemetry.yml`, is there an example in the folder?

Reply:
It can be a new file docker file for thelemetry services using either `telemetry.yml` or `telemetry.cloud.yml` as reference; I made this commit:
https://github.com/graphistry/graphistry-cli/pull/47/commits/e18e5edfff49b469562ed65776f2be1c2b55a96c